bodhi/server/services/overrides.py

```diff
@@ -102,19 +104,19 @@ def query_overrides(request):
     expired = data.get('expired')
     if expired is not None:
         if expired:
-            query = query.filter(BuildrootOverride.expired_date!=None)
+            query = query.filter(BuildrootOverride.expired_date.isnot(None))
```

Comment:
@pypingou Looks like it was introduced in 0.7.9:

http://docs.sqlalchemy.org/en/latest/core/sqlelement.html#sqlalchemy.sql.operators.ColumnOperators.isnot